content/docs/rfcs/38/README.md

```diff
@@ -0,0 +1,59 @@
+---
+slug: 38
+title: 38/LOGOS-CONSENSUS
+name: Initial Logos Consensus Description
+status: raw
+tags: logos
+editor: Mark Evenson <mark.evenson@status.im>
+contributors:
+- Mark Evenson <mark.evenson@status.im>
+---
+# Problematic
+
+      If the doors of perception were cleansed every thing would appear to
+      man as it is: Infinite. For man has closed himself up, till he sees
+      all things thro' narrow chinks of his cavern.
+          -- William Blake "The Marriage of Heaven and Hell"
+          
+      To await and prepare for this moment is to manipulate
+      information not as the last shock wave whose significance
+      escapes everyone, but as the first repercussion of an act still
+      to come.
+          -- Raoul Vaneigm "Basic Banalities"
+      
+
+## In Media Res
+
+We begin--as all worthwhile journeys begin--in the middle of things,
+desiring to communciate a whole without knowing the exact shape of the
+future we have glimpsed in moments of revelation, which while
+untethered from currently existing reality, provides an intoxicating
+fever dream of a more desirable future.
+
+And the dream is this: one can do blockchain without Nakamoto
+consensus.  One can have a leaderless consensus, upon which a
+totalizing view of transactions can be constructed.
+
+
+# Consensus
+# Interoperability 
+## Wire Formats Specification / Syntax
+At this point, we do not commit to a syntax but rather to a semantics
```

Comment:
I'd open a new section (above this one) for the semantics. 

Somboday who just wants to implement, and already has an understanding of how things work, can jump to the sytax section for reference and start implementing.

Reply:
Hmm.  The subtlety is there isn't a wire protocol defined yet, just a semantics of what meanings nodes exchange for consensus rounds.  I was thinking about deriving an interoperable JSON syntax from an abstract semantics, hence this approach.  

I understand and accept your emphasis on ease of implementation here, so will reconsider my approach when I finish getting my PR draft for today ready to push.

Reply:
Wdyt @oskarth @corpetty? Is there any raw material a wire format spec (e.g. protobuf) could be based on?
In case we don't have this, i'd suggest starting with the semantics section and leave the background/intro/motivation for last.
Imo, starting with the semantics section is perfectly fine, too.
Mainly, my point was to put the semantics in a separate section like so:

```

## Theory / Semantics

<If there is no sufficient material to start with a raw wire spec, start here>

## Wire Formats Specification / Syntax

At this point, we do not commit to a syntax but rather to a semantics [...]

```

Reply:
The Theory section has been moved to appendices, being replaced with a complete pseudo-code transcription of Glacier.

For wire formats, users are advised to construct a suitable Waku payload with appropiate metada probaly via JSON, preferrably via JSON-LD.